frontend/cypress/perf/common.ts

```diff
@@ -44,3 +44,41 @@ export function measureListsLoadTime(name: string, visits: number, listUrl: stri
       cy.writeFile(reportFilePath, contents, { flag: 'a+' });
     });
 }
+
+export function measureDetailsLoadTime(name: string, visits: number, detailsUrl: string): void {
+  // Getting an average to smooth out the results.
+  let sum = 0;
+  const visitsArray = Array.from({ length: visits });
+
+  cy.wrap(visitsArray)
+    .each(() => {
+      // Disabling refresh so that we can see how long it takes to load the page without additional requests
+      // being made due to the refresh.
+      cy.visit(detailsUrl, {
+        onBeforeLoad(win) {
+          win.performance.mark('start');
+        }
+      })
+        .its('performance')
+        .then(performance => {
+          cy.get('.pf-v5-c-tabs').should('be.visible');
```

Comment:
Except this line, the rest of the method is exactly equals to `measureListsLoadTime` method. Maybe you can create a generic method `measureLoadTime` with a callback parameter to perform extra validations that are different for list and detail load.